### ISSUE TITLE
Using /inspect on offline players ignores permissions

### ISSUE THREAD

**Issue by TheosisOfficial** [mcMMO-Dev/mcMMO#4399]
Using the `/inspect` command on an offline player will show _all_ their stats instead of just the ones they have permissions for if they haven't joined the server in the current session (I'm assuming this is because their permissions have not been loaded).

It would be nice to see this command work the way it is supposed to, regardless whether or not the target player has joined the server in that session. :)

**Comment by nossr50** [comment]
Bukkit has no built in method to check permissions of an offline player, so we would need to add some kind of library (Vault maybe) to do this and I'm not sure I want to add another library just for this.

**Comment by TheosisOfficial** [comment]
Yeah, you can definitely do it with Vault. I understand that you might not want to import it purely for this though.
